pinot-query-planner/src/test/java/org/apache/pinot/query/QueryEnvironmentTestBase.java

```diff
@@ -130,9 +140,15 @@ protected Object[][] provideQueries() {
 
   public static QueryEnvironment getQueryEnvironment(int reducerPort, int port1, int port2,
       Map<String, Schema> schemaMap, Map<String, List<String>> segmentMap1, Map<String, List<String>> segmentMap2) {
+    return getQueryEnvironment(reducerPort, port1, port2, schemaMap, segmentMap1, segmentMap2, Collections.emptyMap());
```

Comment:
^ Let's add a table or two which have `nullHandlingEnabled: true` in table-config. Anti semi join queries (NOT IN) kind of queries would turn into broadcast joins in that case.